ledger/api-server-damlonx/reference-v2/src/main/scala/com/daml/ledger/api/server/damlonx/reference/v2/Config.scala

```diff
@@ -33,6 +34,7 @@ object Config {
   def default: Config =
     new Config(
       port = 0,
+      address = None,
```

Comment:
If you set the default to `127.0.0.1` or `localhost`, it won't be exposed on a potentially public network interface.

Alternatively you could change the logic here to use `address.getOrElse("localhost")` when constructing the builder instead of letting netty bind to the wildcard address:
https://github.com/digital-asset/daml/blob/52bf6056a4141dff78614908fd26eb14ad1740ef/ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/LedgerApiServer.scala#L90-L91

Reply:
I think it's a good idea to default to localhost; the current behavior is to expose to `0.0.0.0`, which is generally not what we want, especially in development.

Perhaps use `InetAddress.getLoopbackAddress` rather than hard-coding `"localhost"`.

```scala
val host = address
  .map(InetAddress.getByName)
  .getOrElse(InetAddress.getLoopbackAddress)
val socket = new InetSocketAddress(host, port)
val builder = NettyServerBuilder.forAddress(socket)
```

However, this is a breaking change and we should at least add it to the changelog.

Reply:
Yay, once I saw the addess.getOrElse("localhost") I switched off my brain and didn't check where else this is used. So I fixed the PR using a combo of @gerolf-da and @SamirTalwar suggestions. Plus I've added the `CHANGELOG_BEGIN / CHANGELOG_END` section to the commit message.

Reply:
👍